scripts/build-docs.js

```diff
@@ -26,15 +26,17 @@ const parentComponents = _.chain(compoundComponents)
 components.forEach(component => {
   const [componentName, componentParentName] = getComponentNameParts(component.name);
   const isParentComponent = parentComponents.includes(componentName);
+  const isIncubatorComponent = component.category === 'incubator';
 
   let content = '';
+
   /* Markdown Front Matter */
   content += `---\n`;
   if (isParentComponent) {
     content += `sidebar_position: 1\n`;
   }
   content += `id: ${component.name}\n`;
-  content += `title: ${component.name}\n`;
+  content += `title: ${isIncubatorComponent ? 'Incubator.' : ''}${component.name}\n`;
```

Comment:
What does this do? Because I already see `Incubator.Dialog` for example

Reply:
It adds "Incubator" in the component page title. 
So instead of saying "Dialog" it'll say "Incubator.Dialog"